0099-transfer-hotspot.md

```diff
@@ -0,0 +1,72 @@
+# HIPxx: Transfer Hotspot
+
+- Start Date: 2020-09-22
+- Original HIP PR: [#42](https://github.com/helium/HIP/pull/42)
+- Author: [@abhay](https://github.com/abhay)
+- HIP Number: `TODO`
+- Tracking Issue: `TODO`
+
+# Summary
+[summary]: #summary
+
+This proposal introduces a new type of transaction to the Helium blockchain called `transfer_gateway_v1`. This transaction will be an atomic transfer of an onboarded Hotspot / gateway from one owner to another. This transaction can have an optional amount of HNT associated.
+
+# Motivation
+[motivation]: #motivation
+
+Although new RAK Hotspot Miners coming online and future efforts towards other gateways are underway, we can enable existing hotspot owners to redirect coverage resources to hosts who are willing to provide coverage in underutilized areas.
+
+Over 7000 hotspots (at the time of writing) have come online since launching the Helium network. They have been placed in over 1500 cities. Some cities like Modesto, Brooklyn, New York, and San Francisco have over 250 hotspots deployed over a relatively small area. Other notable cities like New Orleans or Knoxville, TN are significantly under covered by existing hotspots.
+
+# Stakeholders
+[stakeholders]: #stakeholders
+
+- Any Helium hotspot owner who has deployed at least one hotspot.
+- The Helium blockchain engineering team.
+
+# Detailed Explanation
+[detailed-explanation]: #detailed-explanation
+
+## Implement a new transaction, `transfer_gateway_v1`
+
+This new transaction would require two parties to sign the transaction in order to update the gateway's owner in the ledger.
+
+### Steps
+
+1. Current owner creates a partially signed transaction with an optional HNT amount that is required to complete the transaction.
+2. Current owner sends the partially signed transaction to the receiving owner.
+3. Recipient signs the transaction and pays the DC fee to submit the transaction to the blockchain.
+4. If the receiving account contains sufficient HNT balance as requested by the current owner and contains enough HNT to burn into DCs for the transaction, the transaction is accepted and the gateway's owner is updated in the ledger.
+5. The hotspot no longer appears in the sender's hotspot list and now appears in the recipient's hotspot list.
+
+## Implement the transaction in the helium-wallet client
+
+`TODO`
```

Comment:
@madninja and I talked about this today and we're not sure we should go into great detail here - the helium-wallet already supports "exporting" a partially completed transaction as either a QR code format or a Base64 encoded text blob - either of these formats would be useful for completing the transaction.

Reply:
That's probably enough information exactly as described. Want to add it or me?

Reply:
Sure thanks